packages/manager/src/views/WebPropertyEnvPage/components/ConfigureAccess/ConfigureAccess.tsx

```diff
@@ -0,0 +1,298 @@
+/* eslint-disable no-underscore-dangle */
+import { useAddPermission, useDeleteMember } from '@app/services/rbac';
+import { toPascalCase } from '@app/utils/toPascalConvert';
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionToggle,
+  Button,
+  Checkbox,
+  Split,
+  SplitItem
+} from '@patternfly/react-core';
+import { TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+import React, { useEffect, useState } from 'react';
+import toast from 'react-hot-toast';
+import { AddDataType, DeleteDataType, GroupDTO, GroupItem } from './types';
+
+type Props = {
+  onClose: () => void;
+  propertyIdentifier: string;
+  memberList: any;
+  flagOpen: boolean;
+};
+export const ConfigureAccess = ({
+  onClose,
+  propertyIdentifier,
+  memberList,
+  flagOpen
+}: Props): JSX.Element => {
+  const [expanded, setExpanded] = useState<string>('def-list-toggle2');
+  const [addAccess, setAddAccess] = useState<string[]>([]);
+  const [deleteAccess, setDeleteAccess] = useState<string[]>([]);
+  const [isShowAdvancedViewEnabled, setIsShowAdvancedViewEnabled] = useState<boolean>(false);
+  const [group, setGroup] = useState<GroupDTO>({ data: memberList?.data });
```

Comment:
Same opinion on group. Why keep this as object if only one key

Reply:
this is the statemutation thing we discussed, added as TODO comment